composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "keboola/db-extractor-common": "^9.2"
+        "keboola/db-extractor-common": "^9.4.0"
```

Comment:
neměl by se přidat i ten logger, když se používá přímo? Předtím byl součástí repa, ne?

Reply:
ano to by měl :( je tam